src/docs/product/releases/health/index.mdx

```diff
@@ -25,7 +25,7 @@ Sentry distinguishes between two kinds of sessions:
 
 <ul>
 
-<li style="list-style-type:none;">A user- or application-mode session begins with the start of the application. Or, it begins with bringing the already started application back from background to the foreground.</li>
+<li style="list-style-type:none;">A user- or application-mode session begins with the start of the application. Or, it begins with bringing the already started application back from background to the foreground. In this context, a user is defined by one or more of the following: a unique ID or UUID, username, email, or IP address.</li>
```

Comment:
just as a sanity check, I'm not sure if any of the SDKs set the ip address here, because the ip address is a separate field in the session. And I'm not sure either how the UI/product side consolidates `did` and `ip_address` while counting sessions.

Here's what javascript sets as `did` (which is the unique session user identifier).
https://github.com/getsentry/sentry-javascript/blob/c518955b774e5bbc36e7f1a22410703469911a2a/packages/hub/src/session.ts#L40-L42

Reply:
Do you think I should just take out IP address here? I've looked at the Python SDK and it seems like it has a similar treatment:
https://github.com/getsentry/sentry-python/blob/23e02952e41321e19303fcc0b1c29b4ed15a9ecc/sentry_sdk/session.py#L103-L104

Reply:
yea let's remove ip address